packages/Webkul/Shop/src/Resources/views/checkout/onepage/summary.blade.php

```diff
@@ -29,21 +29,20 @@ class="flex gap-x-[15px] pb-[20px]"
                     >
                         <img
                             class="max-w-[90px] max-h-[90px] w-[90px] h-[90px] rounded-md"
-                            :src="item.base_image.small_image_url"
+                            :src="item.base_image.medium_image_url"
+                            :title="item.name"
                             :alt="item.name"
-                            width="110"
-                            height="110"
                         />
 
                         <div>
                             <p 
                                 class="text-[16px] text-navyBlue max-sm:text-[14px] max-sm:font-medium" 
-                                v-text="item.name"
+                                v-html="item.name"
                             >
                             </p>
 
                             <p class="mt-[10px] text-[18px] font-medium max-sm:text-[14px] max-sm:font-normal">
-                                @{{ item.formatted_price }} X @{{ item.quantity }}
+                                <span v-html="item.formatted_price"> </span> X @{{ item.quantity }}
```

Comment:
Also remove the `v-html` fro everywhere, we are not giving formating at this stage.

Reply:
Just add left and right switch codes, don't add any HTML that needs to change the v-html.

Reply:
Yeah, i know, so we are not giving any styling at this stage. Just remove the last pointer which doing formating and stylings.

Reply:
Styling has been removed, and the files where 'v-html' and {!! !!} were used have been restored to their previous state.